tests/conftest.py

```diff
@@ -120,8 +124,10 @@ def config_data():
 
 
 @pytest.fixture()
-def local_client_mock():
-    return Mock()
+def local_client_mock(config_data):
+    mock = Mock()
+    mock.config_parser = ConfigParser(config_data)
+    return mock
```

Comment:
not need as I commented above. I'll create full local client mock if some tests not related with local_client will need it